src/Blockcore.Indexer/Storage/Types/SyncRichListInputs.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Blockcore.Indexer.Storage.Types
+{
+   class SyncRichListInputs
+   {
+      public long value { get; set; }
```

Comment:
All public should follow PascalCase, as according to .NET Naming guidelines. This should be "Value". private fields should be "value".